pkg/operator/rbaccontroller/rbaccontroller.go

```diff
@@ -0,0 +1,90 @@
+package rbaccontroller
+
+import (
+	"context"
+	"strings"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	errors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// Bug 1821771: We needed a way to tighten the cluster role binding when cluster
+	// is upgraded from 4.1
+
+	// openShiftManagedRoleBindingAnnotation is a cluster role binding annotation used
+	// by operator to manage it when set to true
+	openShiftManagedRoleBindingAnnotation = "openshiftapiserver.rbac.authorization.openshift.io/managed"
+)
+
+type RBACController struct {
```

Comment:
document what this does

Reply:
Done! please check and let me know if it needs changes

Reply:
"RBACController this controller" -> "RBACController removes..."

But same applies as for the annotation comment: it must be generic

Reply:
RBACReconcileController would be more specific.